client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Microsoft.AspNetCore.SignalR.Client.TS.csproj

```diff
@@ -30,26 +33,30 @@
     <RemoveDir Directories="$(SignalRClientDistFolder)" />
   </Target>
 
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
 
   <Target Name="Pack">
     <Message Text="Creating TypeScript client Node module" Importance="High" />
-    <Exec Command="node -pe &quot;require('./package.json').version&quot;" ConsoleToMSBuild="true" WorkingDirectory="$(MSBuildThisFileDirectory)..">
-      <Output TaskParameter="ConsoleOutput" ItemName="ExecOutput" />
-    </Exec>
+
     <PropertyGroup>
-      <ModuleVersion>@(ExecOutput)</ModuleVersion>
-      <ModuleVersion Condition="'$(VersionSuffix)' != ''">$(ModuleVersion)-$(VersionSuffix)</ModuleVersion>
       <IsGitRepository Condition="EXISTS('$(MSBuildThisFileDirectory)..\..\.git\HEAD')">true</IsGitRepository>
       <IsGitRepository Condition="'$(IsGitRepository)'==''">false</IsGitRepository>
     </PropertyGroup>
-    <Exec Command="npm version $(ModuleVersion)" WorkingDirectory="$(MSBuildThisFileDirectory).." Condition="'$(VersionSuffix)' != ''" />
-    <Exec Command="npm pack" WorkingDirectory="$(MSBuildThisFileDirectory).." />
-    <Exec Command="git checkout HEAD -- &quot;$(MSBuildThisFileDirectory)../package.json&quot;" Condition="$(IsGitRepository) AND '$(VersionSuffix)' != ''" />
-    <Exec Command="git checkout HEAD -- &quot;$(MSBuildThisFileDirectory)../package-lock.json&quot;" Condition="$(IsGitRepository) AND '$(VersionSuffix)' != ''" />
+
+    <Exec Command="npm version $(PackageVersion) --allow-same-version" WorkingDirectory="$(MSBuildThisFileDirectory)..\" />
```

Comment:
@anurse one thing I noticed from the macOS build is this new command-line flag is not available with node 6. Any reason not to require node 8 to build?

Reply:
not that I can think of, node 8 should be fine.